bolt/include/bolt/Core/BinaryFunction.h

```diff
@@ -1885,6 +1890,42 @@ class BinaryFunction {
     return LSDASymbols[F.get()];
   }
 
+  /// If all landing pads for the function fragment \p F are located in fragment
+  /// \p LPF, designate \p LPF as a landing-pad fragment for \p F. Passing
+  /// std::nullopt in LPF, means that landing pads for \p F are located in more
+  /// than one fragment.
+  void setLPFragment(const FragmentNum F, std::optional<FragmentNum> LPF) {
+    if (F.get() >= LPFragments.size())
```

Comment:
Probably I'm missing something here -- but why resize + [] instead of insert?

Reply:
Because we are growing the vector on demand? Now I'm thinking if I'm missing something.

Reply:
SG, nevermind.